web/frontend/src/components/slam-2d-render.vue

```diff
@@ -7,6 +7,42 @@ import * as THREE from 'three';
 import { MapControls } from 'three/examples/jsm/controls/OrbitControls';
 import { PCDLoader } from 'three/examples/jsm/loaders/PCDLoader';
 import type { commonApi } from '@viamrobotics/sdk';
+import { SVGLoader } from 'three/examples/jsm/loaders/SVGLoader';
+import { baseMarkerUrl } from '../lib/base-marker-url';
+import { destMarkerUrl } from '../lib/destination-marker-url';
+
+type SvgOffset = {
+  x: number,
+  y: number,
+  z: number
+}
+
+const baseMarkerOffset: SvgOffset = {
+  x: 0.35,
+  y: 0,
+  z: 0,
+};
+
+const destinationMarkerOffset: SvgOffset = {
+  x: 1.2,
+  y: 2.5,
+  z: 0,
+};
+
+const backgroundGridColor = 0xCA_CA_CA;
+
+const gridSubparts = ['AxesPos', 'AxesNeg', 'Grid'];
+
+const gridHelperRenderOrder = 997;
+const axesHelperRenderOrder = 998;
+const svgMarkerRenderOrder = 999;
+
+const gridHelperScalar = 4;
+const axesHelperSize = 8;
+
+// Note: updating the scale of the destination or base marker requires an offset update
```

Comment:
[opt] I would move lines 20-30 to under this comment so it explains both the offset and scale

Reply:
kk